src/main/java/org/example/service/ConnectionService.java

```diff
@@ -0,0 +1,57 @@
+package org.example.service;
+
+import java.sql.*;
+
+
+public class ConnectionService {
+
+    private static final String DATABASE_NAME = "postgres";
+    private static final String DATABASE_USER = "postgres";
+    private static final String DATABASE_PASSWORD = "postgres";
+    private static final String DRIVER_POSTGRESQL = "org.postgresql.Driver";
+    private static final String JDBC_URL = "jdbc:postgresql://developers-db:5432/";
```

Comment:
Where the `JDBC_URL` variable is used?

Reply:
nowhere it's just an enxample for me I forgot to delete it

Reply:
fixed